src/_pylibmcmodule.c

```diff
@@ -2491,9 +2554,14 @@ static void _make_excs(PyObject *module) {
     PylibMCExc_Error = PyErr_NewException(
             "pylibmc.Error", NULL, NULL);
 
+    PylibMCExc_CacheMiss = PyErr_NewException(
```

Comment:
Should it inherit from `BaseException` instead of `Exception`? This seems [not recommended](https://docs.python.org/2/library/exceptions.html).

For comparison, `StopIteration` inherits from `Exception` (but not from `StandardError`).